src/components/app/list-new/HelmAppList.tsx

```diff
@@ -393,7 +398,14 @@ export default function HelmAppList({
                     <span className="app-list__cell-header">{APP_LIST_HEADERS.Namespace}</span>
                 </div>
                 <div className="app-list__cell app-list__cell--time">
-                    <span className="app-list__cell-header">{APP_LIST_HEADERS.LastDeployedAt}</span>
+                    <button className="app-list__cell-header flex" onClick={sortByLastDeployed}>
```

Comment:
Why this button, can't it be done via some span?

Reply:
done